### PR TITLE
[change] Switch to iwinfo to get associated clients if mesh or adhoc

### PR DESCRIPTION
**Pull request by R9295**
implements #21

### CODE REVIEW

**Review by nemesisdesign — COMMENTED** [review]
Thanks @R9295! It looks good.
I need some more time to test it, but before doing that, let me know if you have tested it on openwisp-monitoring and if something needs to be done in https://github.com/openwisp/openwisp-monitoring/issues/260 to make it work there.

**Comment by R9295** [comment]
Will do

**Review by nemesisdesign — COMMENTED** [review]
Great work @R9295 :+1: 

I tested it with openwisp-monitoring, the schema needs to be relaxed a bit:

```
"Invalid data in \"#/interfaces/4/wireless/clients/0\", validator says:\n\n'aid' is a required property"r
```

Can you take care of this change? (https://github.com/openwisp/openwisp-monitoring/issues/260)

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+			
+		$(INSTALL_BIN) \
+			files/monitoring-crontab \
+			$(1)/etc/crontabs/root
+    			
+
+endef
+
+
+$(eval $(call BuildPackage,openwrt-netjson-monitoring))
```
Comment (nemesisdesign) [inline Makefile:53]: Let's not add this now, it's not related to the issue we're working on

File: files/monitoring-crontab [hunk 2]
```diff
@@ -0,0 +1 @@
+  */5 * * * * /usr/sbin/openwisp-monitoring
```
Comment (nemesisdesign) [inline files/monitoring-crontab:1]: same here

**Review by nemesisdesign — CHANGES_REQUESTED** [review]

**Comment by R9295** [comment]
@nemesisdesign whoops, I accidentally pushed that to master and not my dev branch 😳

**Comment by nemesisdesign** [comment]
> @nemesisdesign whoops, I accidentally pushed that to master and not my dev branch

Ah ok, I suspected that, no problem.

**Review by nemesisdesign — COMMENTED** [review]
Tested with https://github.com/openwisp/openwisp-monitoring/pull/263 and it works, thanks! :+1: :+1: 
I will wait for you to clean up the mistakenly added commit before merging.

**Review by nemesisdesign — APPROVED** [review]
Great work @R9295, thanks!

**Pull request merged**